--- User-Utmp-1.01/README.orig	Sat Aug  3 12:54:23 2002
+++ User-Utmp-1.01/README	Sat Aug  3 12:55:56 2002
@@ -9,7 +9,7 @@
 -----------
 
 This module provides a simple Perl interface to utmp- and utmpx-style
-databases on UNIX systems, the most important being /etc/utmp, which
+databases on UNIX systems, the most important being /var/run/utmp, which
 provides information about users currently logged in.  There is also
 experimental support for writing utmp files.  For further details see
 the POD documentation embedded in the file Utmp.pm, which will also be
--- User-Utmp-1.01/Utmp.pm.orig	Sat Aug  3 12:50:34 2002
+++ User-Utmp-1.01/Utmp.pm	Sat Aug  3 12:54:13 2002
@@ -88,7 +88,7 @@
 Reads a utmp-like file and converts it to a Perl array of hashes.
 Each array element (a reference to a hash) represents one utmp record.
 The hash keys are the names of the elements of the utmp structure as
-described in utmp(4).  The hash values are the same as in C.
+described in utmp(5).  The hash values are the same as in C.
 
 Note that even if C<ut_addr> (if provided by the utmp implementation)
 is declared as I<long>, it contains an Internet address (four bytes in
@@ -101,8 +101,8 @@
 Reads a utmpx-like file and converts it to a Perl array of hashes.
 Each array element (a reference to a hash) represents one utmp record.
 The hash keys are the names of the elements of the utmpx structure as
-described in utmpx(4) or getutx(3).  The hash values are the same as
-in C.
+described in utmpx(5) or getutxent(3).  The hash values are the same
+as in C.
 
 Note that even if C<ut_addr> (if provided by the utmpx implementation)
 is declared as I<long>, it contains an Internet address (four bytes in
@@ -120,9 +120,9 @@
 =item B<utmpname()>
 
 Allows the user to change the name of the file being examined from the
-default file (typically /etc/utmp or, for utmpx, /etc/utmpx) to any
-other file.  In this case, the name provided to utmpname() will be
-used for the getut(), getutx(), and putut() functions.
+default file (typically /var/run/utmp or, for utmpx, /var/run/utmpx)
+to any other file.  In this case, the name provided to utmpname() will
+be used for the getut(), getutx(), and putut() functions.
 
 =back
 
@@ -140,7 +140,7 @@
 =head1 RESTRICTIONS
 
 Reading the whole file into an array might not be the most efficient
-approach for potentially large files like /etc/wtmp.
+approach for potentially large files like /var/log/wtmp.
 
 This module is based on the traditional, non-reentrant utmp functions;
 it is therefore B<not> thread-safe.
@@ -151,6 +151,6 @@
 
 =head1 SEE ALSO
 
-utmp(4), getut(3), utmpx(4), getutx(3)
+utmp(5), getutent(3), utmpx(5), getutxent(3)
 
 =cut
--- User-Utmp-1.01/example.pl.orig	Sat Aug  3 12:54:30 2002
+++ User-Utmp-1.01/example.pl	Sat Aug  3 12:55:30 2002
@@ -5,10 +5,10 @@
 use Socket;
 
 # This is typically the default utmp file
-# User::Utmp::utmpname("/etc/utmp");
+# User::Utmp::utmpname("/var/run/utmp");
 
 # And this is a typical alternative
-# User::Utmp::utmpname("/var/adm/wtmp");
+# User::Utmp::utmpname("/var/log/wtmp");
 # Get the contents of the utmp file
 @utmp = User::Utmp::getut();
 
@@ -59,10 +59,10 @@
 print "#" x 79, "\n", "Utmpx", "\n";
 
 # This is typically the default utmpx file
-# User::Utmp::utmpname("/etc/utmpx");
+# User::Utmp::utmpname("/var/run/utmpx");
 
 # And this is a typical alternative
-# User::Utmp::utmpname("/var/adm/wtmp");
+# User::Utmp::utmpname("/var/log/wtmp");
 # Get the contents of the utmpx file
 @utmpx = User::Utmp::getutx();
 
